spec/game_looper_spec.rb

```diff
@@ -1,4 +1,14 @@
 require 'game_looper'
+require 'console'
+require 'display'
+require 'prompt'
+require 'board'
+require 'player'
+require 'number_validator'
+require 'outcome_checker'
+require 'ui'
+require 'stringio'
+require 'pry'
```

Comment:
Try not to commit debugging tools like `pry` or print statements. I sometimes let them slip too.